sdk/openai/Azure.AI.OpenAI/tests/Samples/Sample04_StreamingChat.cs

```diff
@@ -29,18 +27,18 @@ public async Task StreamingChatWithNonAzureOpenAI()
                 }
             };
 
```

Comment:
Would be very appreciated a `FunctionCall` streaming example, 

How can we manage the streaming message content with the message.FunctionCall content in a streamlined fashion.

Reply:
That's a great suggestion! We should have had this even *before* changing things; having one now seems critical. I added a snippet that demonstrates the streaming accumulation approach as applied to function calling.